docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/cpu-device.rst

```diff
@@ -96,6 +99,7 @@ CPU plugin supports the following floating-point data types as inference precisi
 - ``f32`` (Intel® x86-64, Arm®)
 - ``bf16`` (Intel® x86-64)
 - ``f16`` (Intel® x86-64, Arm®)
+- ``MXFP4`` (Intel® x86-64)
```

Comment:
actually mxfp4 is not the inference precision on CPU plugin. suggest to remove this line.

Reply:
I have removed it in this commit: [4946e7154b8d677093c63f548bbf9c3bcb1fe551](https://github.com/openvinotoolkit/openvino/pull/26554/commits/4946e7154b8d677093c63f548bbf9c3bcb1fe551)